### DISCUSSION TITLE
Get a changelog by day

### DISCUSSION THREAD

**Discussion by arnaudambro** [semantic-release/semantic-release#2220] [Q&A]
## New feature motivation

The changelog is made by version now, but in our project we don't care that much about versions, and we want to communicate the changelog with some non-devs who would be much more interested in having the changelog by grouping features/fixes by day rather than by version, is it possible ?

**Comment by travi** [comment]
the purpose of semantic-release is to simplify versioning of your project semantically. a release driven by release determines the new version during the same process that would update your changelog, so the updates are directly tied to creating the new _version_ and are not tied to a time period like the day. 

semantic-release is not intended to be a generic workflow tool, and it sounds like semantic-release may not be the tool you are looking for to fit the process you are trying to manage.
